### PR TITLE
Update hardware and debug fields

### PR DESCRIPTION
**Pull request by haslinghuis**
Update debug, gyro, acc and baro definitions.
Also rebased this file a bit.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — DISMISSED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
